### ISSUE TITLE
siri commands

### ISSUE THREAD

**Issue by pmcochrane** [PeteLawrence/homebridge-people#17]
Any idea what Siri commands are available for the plugin? I can get the Eve app to show my occupancy but cannot get Siri to recognise the device.

Thanks

**Comment by PeteLawrence** [comment]
I don't believe that there are any Siri commands yet for occupancy sensors.  I'll leave this issue open though incase anyone wants to correct me, or so that when support is added any watchers can be notified.

**Comment by nitaybz** [comment]
you can ask siri: "Whats the status of [your sensor name]?"
when she said: "your "[your sensor name]" goes off" that means it is triggered.
